### ISSUE TITLE
Synpop and Abpop in pymoogi

### ISSUE THREAD

**Issue by sakaricm** [madamow/pymoogi#19]
Hi Monika, thanks for all your hard work with Pymoogi!  I've noticed that Synpop and Abpop aren't included in Pymoogi.  Would it be possible to get those drivers added back in?  Thanks!

**Comment by madamow** [comment]
Synpop and Abpop were never included in pymoogi. The reason for that is I have never used them in my work. If you can  provide an example input data and .par files for those drivers, I can try adding them to pymoogi.

**Comment by sakaricm** [comment]
Great, I can send you a set of files and instructions.  Are you still at UT Austin?  Thanks!

**Comment by madamow** [comment]
No, right now I am with University of Illinois at Urbana-Champaign.
